Quadriga/src/main/java/edu/asu/spring/quadriga/web/workbench/ConceptCollectionProjectController.java

```diff
@@ -66,6 +70,33 @@ public String listProjectConceptCollection(@PathVariable("projectid") String pro
 		return "auth/workbench/project/conceptcollections";
 	}
 
+
+	@RequestMapping(value = "auth/workbench/{projectid}/conceptcollectionsJson", method = RequestMethod.GET, produces = "application/json")
+	@ResponseBody
+	public String listProjectConceptCollectionJson(@PathVariable("projectid") String projectid, Model model) throws QuadrigaStorageException, JSONException {

```

Comment:
method should return a responseentity https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/http/ResponseEntity.html